packages/code-du-travail-data/dataset/outils.json

```diff
@@ -43,18 +35,10 @@
       "cout embauche CDI"
     ],
     "description": "Estimez le salaire lors d'une embauche : total employeur, brut, net",
```

Comment:
we should ask what description / text we should keep for each tools, for now there is one in the `outils.json`, one `tools/[slug].js`